Source/Atomic/Graphics/AnimatedModel.cpp

```diff
@@ -1044,7 +1044,7 @@ void AnimatedModel::OnWorldBoundingBoxUpdate()
     if (isMaster_)
     {
         // Note: do not update bone bounding box here, instead do it in either of the threaded updates
-        worldBoundingBox_ = boneBoundingBox_.Transformed(node_->GetWorldTransform());
+        worldBoundingBox_ = boundingBox_.Transformed(node_->GetWorldTransform());
     }

```

Comment:
Ok, I understand the issue, unfortunately this will break the runtime bounding box, potentially causing false clipping, etc.

Please check for the editor context:  ```context_->GetEditorContext();``` before using the boundingBox instead, also place in an // ATOMIC BEGIN and // ATOMIC END block, with either a filed issue linked or as best and concise of a description as you can muster.  Otherwise, when pulling upstream, these changes become very easy to lose/tell if they are still needed.  

This issue will go away once prefabs can be revisited.